### DISCUSSION TITLE
Which "languages" are available for Fenced Code highlighting

### DISCUSSION THREAD

**Discussion by irmod** [damionx7/Zettel-Notes-Documentation#210] [General]
@damionx7, tell me which "languages" are available for Fenced Code highlighting

````
```language
code
code
code
```
````

**Comment by damionx7** [comment]
Hey, syntax highlighting is support for following languages.

```
bash / zsh / sh
brainf*ck
c
clike
clojure
cpp
csharp (dotnet)
css (+css-extras)
dart
git
go
groovy (no string interpolation)
java
javascript (js)
json (jsonp)
kotlin
latex
makefile
markdown
markup (xml, html, mathml, svg)
python
scala
sql
swift
yaml
```

**Comment by irmod** [comment]
@damionx7, what do you use in Zettel Notes for syntax highlighting (CodeMirror, Ace Editor, Monaco Editor, Prism...)?

**Reply by damionx7** [reply]
I use prism4j

**Comment by irmod** [comment]
@damionx7, thanks for the information... I'm a little confused about the spelling of the language "markdown" or "md"...

**Reply by damionx7** [reply]
Whats the confusion :thinking:

**Comment by irmod** [comment]
@damionx7, I use it a lot now and have a lot of notes where the Fenced Code Language is defined as "md".

Could you, in accordance with the documentation for the **prism4j** library, add "md" `@Aliases` for "markdown" (like "js" for "javascript") to Zettel Notes.

<https://github.com/noties/Prism4j#contributing>

> `@Aliases` specifies what aliases a language has. For example `markup` language has these: `@Aliases({"html", "xml", "mathml", "svg"})`. So when a `GrammarLocator` will be asked for a `svg` language the `markup` will be returned.
> 
> ```
> @Aliases({"html", "xml", "mathml", "svg"})
> public class Prism_markup {}
> ```

**Reply by damionx7** [reply]
Sure, I will add it.
